_data/sidebars/home_sidebar.yml

```diff
@@ -54,7 +54,7 @@ entries:
       url: /gestures/
       output: web
 
-    - title: Layouts
+    - title: Box Constraints
```

Comment:
We might not need to put it in the sidebar if this is a doc for a limited audience.